src/main.rs

```diff
@@ -290,14 +295,14 @@ impl HabitCtl {
 
             for habit in self.get_todo(&current) {
                 self.print_habit_row(&habit, log_from.date(), current.clone());
-                let l = format!("[y/n/-] ");
+                let l = format!("[y/n/s/⏎] ");
```

Comment:
That ⏎ is much more informative! Might there not be issues with rendering if it isn't included in the terminal's font, though?

Reply:
I believe that may be possible, but afaik (which is not much) I thought the ⏎ is standard in all Unicode fonts, just rendered differently depending on the font in question. Technically, you are correct, it might be missing, but I honestly did not think anyone used non-unicode fonts anymore these days on the erminal. I guess there may be some ttf/otf fonts being used but I believe they all have his keyboard character (ok, guessing not belivee.). Fair call though. Leave it in and see who screams (if anyone)?

Reply:
You're right, it's most likely fine :)